### ISSUE TITLE
Command /vote problem

### ISSUE THREAD

**Issue by Iznogoud76** [BenCodez/VotingPlugin#260]
Hi. Voting plugin is set to automatically create links for voting sites but when I type /vote it returns me a message  "vote here: www.votelinkhere.com". How to add links to voting sites to work with this command?

**Comment by BenCodez** [comment]
The plugin doesn't automatically set the. VoteURL, look in votesites.yml

**Comment by Iznogoud76** [comment]
I have it in VoteSites.yml but it not work for /vote command... 
Here is my VoteSites.yml: https://pastebin.com/aGN2kFDP
Is there something else to set up for that command to work?

**Comment by BenCodez** [comment]
Try doing it like this:
VoteURL: 'URL'

Also did you /av reload?

**Comment by Iznogoud76** [comment]
Nope. Its not working. I tried with "url" too but nothing... Yes, I do /av reload every time.

**Comment by BenCodez** [comment]
You have a yml error, try this:
https://pastebin.com/fa2Mqzdb

**Comment by Iznogoud76** [comment]
Nothing... Same output for /vote command... When I do /vgui everything is fine, I see all links under URL section but with /vote command still not working.

**Comment by BenCodez** [comment]
Restart the server and give me the startup log.

**Comment by Iznogoud76** [comment]
[latest.zip](https://github.com/Ben12345rocks/VotingPlugin/files/1286432/latest.zip)

**Comment by BenCodez** [comment]
Can you send me a screenshot of the problem?

**Comment by Iznogoud76** [comment]
![2017-09-08_02 38 02](https://user-images.githubusercontent.com/31401422/30191075-dbd1b40a-943e-11e7-9d1b-81fdc1fe3a54.png)

**Comment by BenCodez** [comment]
That looks like a different plugin doing that.

**Comment by Iznogoud76** [comment]
Damn. I found it... Venture chat have same command for votelinks :)
Now it works, when I type /vote it opens GUI...
Thank you for your time and I'm sorry if i bother you. I did not even think that another plugin was the cause of the problem when I saw that the 'votelinks' was mentioned in the message :)
Thanks again.

P.S.
Any chance to port this great, great plugin to spongeforge? It would be awesome.
